src/providers/reka-ai/chatComplete.ts

```diff
@@ -0,0 +1,107 @@
+import { REKA_AI } from '../../globals';
+import { Params } from '../../types/requestBody';
+import {
+  ChatCompletionResponse,
+  ErrorResponse,
+  ProviderConfig,
+} from '../types';
+import {
+  generateErrorResponse,
+  generateInvalidProviderResponseError,
+} from '../utils';
+
+export const RekaAIChatCompleteConfig: ProviderConfig = {
+  model: {
+    param: 'model_name',
+    required: true,
+    default: 'reka-flash',
+  },
+  messages: {
+    param: 'conversation_history',
+    transform: (params: Params) => {
+      const mappedMessages = params.messages?.map((message) => ({
+        type: message.role === 'user' ? 'human' : 'model',
+        text: message.content,
+      }));
+      return mappedMessages;
+    },
+  },
+  max_tokens: {
+    param: 'request_output_len',
+  },
+  temperature: {
+    param: 'temperature'
+
+  },
+  top_p: {
+    param: 'runtime_top_p',
+  },
+  top_k: {
+    param: 'runtime_top_k',
+  },
+  stop: {
+    param: 'stop_words',
+  },
+
```

Comment:
Their [docs](https://docs.reka.ai/http-api.html#generation) have more options, but I wasn't sure how to map them here. What do you think would be the best mapping?